docs/simplification.md

```diff
@@ -12,89 +18,83 @@ The AsyncAPI Model SDK supports simplification of:
 - [Required](#determining-the-required-properties-for-the-model)
 - [Extend](#determining-the-extend-for-the-model)
 
+<br/>
+
 ## Determining the type for the model
 
-In order to determine all the possible types a schema can be, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
+In order to determine all the possible types a model can be based on schema, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
 
-### Precedence
+**Precedence of JSON Schema keywords**
 
 The precedence of keywords are in which order we infer or determine types in. Notice it goes form left to right meaning first we set the type using the `type` keyword and then it moves right applying them one by one. The following are the precedence for determining types:
 
-`type` --> `allOf` --> `oneOf` --> `anyOf` --> `then` --> `else` --> `enum` --> `const` --> `not`
+<p align="center">type --> allOf --> oneOf --> anyOf --> then --> else --> enum --> const --> not</p>
 
-#### Absence of data type
+**Absence of data type**
 
-If a schema is defined as `true` or `false` we infer all possible JSON types.
+If a schema is defined as `true` or `false` we infer all possible JSON Schema types.
 
-#### Enum and const
+**Enum and const**
 
 `enum` and `const` are ONLY used to infer the type if it has not already been defined. Type is inferred from the provided value(s).
 
-#### Not
-
-`not` if defined always overwrites any existing inference of types.
+**Not**
 
-#### Non-mentioned keywords
+if defined removes any defined/inferred types defined in `not`.
 
-Any keywords not mentioned as a title are all cumulating the type.
+<br/>
 
 ## Determining the enums for the model
 
-In order to determine all the possible enums a schema can be we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords and in which order they are applied.
+In order to determine the possible enums a model can be we both infer and use existing definitions, however we need to define a precedence for JSON Schema keywords and in which order they are applied.
```

Comment:
Same as https://github.com/asyncapi/generator-model-sdk/pull/87/files#r580500164. I feel it sounds a bit redundant.